libs/glfw/build.zig

```diff
@@ -67,29 +69,39 @@ pub const pkg = std.build.Pkg{
 };
 
 pub fn link(b: *Builder, step: *std.build.LibExeObjStep, options: Options) void {
-    const lib = buildLibrary(b, step, options);
+    const lib = buildLibrary(b, step.build_mode, step.target, options);
     step.linkLibrary(lib);
     addGLFWIncludes(step);
-    if (!options.shared) linkGLFWDependencies(b, step, options);
-    if (options.shared) step.defineCMacro("GLFW_DLL", null);
+    if (options.shared) {
+        step.defineCMacro("GLFW_DLL", null);
+        // TODO(build-system): pass system SDK options through
+        system_sdk.include(b, step, .{});
```

Comment:
This looks like a change in behavior from before, but I'm not sure what the motivation behind it is?

Reply:
we still need header files. there's no linking against sdk libs